src/main/java/saber/parser/SaberParser.java

```diff
@@ -0,0 +1,175 @@
+package saber.parser;
+
+/** Import all saber commands as the parse function require all of them */
+import saber.Saber;
+import saber.commands.*;
+
+import saber.exceptions.MissingArgumentException;
+import saber.exceptions.MissingTimeException;
+import saber.exceptions.SaberCommandNotFoundException;
+
+/** A class that encapsulates a parser for the Saber application (to parse user commands) */
```

Comment:
Class javadocs should start on a new line after `/**`
```suggestion
/** 
 * A class that encapsulates a parser for the Saber application (to parse user commands) 
 */
```

Reply:
Ok